spec/resources/workbooks_spec.rb

```diff
@@ -201,6 +221,17 @@ def find_or_publish_workbook(name)
     end
   end
 
+  # http://onlinehelp.tableau.com/v9.0/api/rest_api/en-us/help.htm#REST/rest_api_ref.htm#Update_Workbook%3FTocPath%3DAPI%2520Reference%7C_____59
+  describe '.update' do
```

Comment:
Sorry, I just noticed the names are mismatched (`.` and `#`). They're all instance methods, so should all be `#`.